pfrl/explorers/additive_gaussian.py

```diff
@@ -23,6 +23,14 @@ def __init__(self, scale, low=None, high=None):
         self.low = low
         self.high = high
 
+    def distribution(self, x):
+        base_distribution = torch.distributions.Independent(
+            torch.distributions.Normal(loc=x, scale=self.scale), 1
+        )
+        return torch.distributions.transformed_distribution.TransformedDistribution(
+            base_distribution, [torch.distributions.transforms.TanhTransform(cache_size=1)]
```

Comment:
this feels a bit confusing because the standard gaussian doesn't use tanh transform, right?

Reply:
you are correct. One thing I observed was that, the resulting kl divergence becomes the shape of [batch_size, action_size] without transformedDistribution. Any idea?

Reply:
interesting, I don't know why lol